config.yml.sample

```diff
@@ -1,5 +1,6 @@
 development:
   cookie_secret: sample
+  cookie_ssl_only: false
```

Comment:
nope. that's the main one; `vagrant up` would not result in a functional VM.

(no public DNS means no ability to procure a letsencrypt cert; no cert means nginx will balk at spinning up the SSL server; balking at spinning up the SSL server means nginx will not start at all; nginx not starting means ansible will not complete execution.)